www-apps/redmine/redmine-3.4.12.ebuild

```diff
@@ -0,0 +1,232 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+USE_RUBY="ruby23 ruby24"
+inherit eutils depend.apache ruby-ng
+
+DESCRIPTION="Flexible project management web application using the Ruby on Rails framework"
+HOMEPAGE="https://www.redmine.org/"
+SRC_URI="https://www.redmine.org/releases/${P}.tar.gz"
+
+KEYWORDS="~amd64"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="imagemagick fastcgi ldap markdown mysql passenger postgres sqlite"
+
+ruby_add_rdepend "
+	fastcgi? ( dev-ruby/fcgi )
```

Comment:
This is looking better, just move the conditional block below unconditional ones.